crates/ruff/src/rules/pydocstyle/rules/blank_before_after_class.rs

```diff
@@ -228,8 +228,33 @@ pub(crate) fn blank_before_after_class(checker: &mut Checker, docstring: &Docstr
         }
 
         let mut lines = UniversalNewlineIterator::with_offset(after, after_range.start());
+        let first_line = lines.next();
+        let first_line_start = first_line.as_ref().map(|l| l.start()).unwrap_or_default();
+
+        // Edge case: The next statement after the docstring is on the same line separated by a
+        // semicolon.
+        if let Some(first_line) = &first_line {
```

Comment:
We have some has_following_content utilities - can we use any of them here?

Reply:
i tried but i don't think they fit here